### PR TITLE
[9.0][hr_timesheet_activity_begin_end] migration to v9

### PR DESCRIPTION
**Pull request by Martronic-SA**
model hr.analytic.timesheet has been removed so we have moved functions to account.analytic.line and changed model of related views.

### CODE REVIEW

**Comment by Martronic-SA** [comment]
@api.one deprecated? really?

**Comment by leemannd** [comment]
@Martronic-SA Yes since V9.0 https://www.odoo.com/documentation/9.0/reference/orm.html

File: hr_timesheet_activity_begin_end/__openerp__.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # -*- coding: utf-8 -*-
 #
 #
-#    Authors: Guewen Baconnier
+#    Authors: Guewen Baconnier, Fabien Moret
 #    Copyright 2015 Camptocamp SA
+#    Copyright 2017 Martronic SA
 #
 #    This program is free software: you can redistribute it and/or modify
 #    it under the terms of the GNU Affero General Public License as
```
Comment (leemannd) [inline hr_timesheet_activity_begin_end/__openerp__.py]: Short headers

File: hr_timesheet_activity_begin_end/__openerp__.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # -*- coding: utf-8 -*-
 #
 #
-#    Authors: Guewen Baconnier
+#    Authors: Guewen Baconnier, Fabien Moret
```
Comment (leemannd) [inline hr_timesheet_activity_begin_end/__openerp__.py]: Can be removed

File: hr_timesheet_activity_begin_end/models/account_analytic_line.py [hunk 3]
```diff
@@ -44,9 +44,10 @@ class AccountAnalyticLine(models.Model):
     time_start = fields.Float(string='Begin Hour')
```
Comment (leemannd) [inline hr_timesheet_activity_begin_end/models/account_analytic_line.py:28]: On the top of the file:
- short header
s/openerp/odoo

File: hr_timesheet_activity_begin_end/views/hr_analytic_timesheet.xml [hunk 4]
```diff
@@ -3,7 +3,7 @@
   <data>
```
Comment (leemannd) [inline hr_timesheet_activity_begin_end/views/hr_analytic_timesheet.xml]: To be removed

**Review by leemannd — CHANGES_REQUESTED** [review]
Thanks for the port.
Could you please update to the latest OCA guidelines?
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0 and projet template https://github.com/OCA/maintainer-tools/tree/master/template/module

**Comment by vvrossem** [comment]
FYI @Martronic-SA @leemannd, we've been using this module since several weeks, and it works smoothly :)

**Review by robinkeunen — APPROVED** [review]

**Comment by leemannd** [comment]
@vvrossem My comment has been done as non blocking.
IMO: I can review it back once it passes travais.

**Comment by houssine78** [comment]
@leemannd I think it passed travis successfully

File: hr_timesheet_activity_begin_end/models/account_analytic_line.py [hunk 5]
```diff
@@ -1,23 +1,7 @@
...
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-#
+# © 2015 Camptocamp SA (author: Guewen Baconnier)
+# © 2017 Martronic SA
```
Comment (leemannd) [inline hr_timesheet_activity_begin_end/models/account_analytic_line.py]: According to https://github.com/OCA/maintainer-tools/pull/197 we should use `Copyright`.

Non-blocking comment

**Review by leemannd — APPROVED** [review]
LGTM. Code review only

**Comment by leemannd** [comment]
For future reviewers, I'm not in favour of having changes imposed like change to use the new description. IMO this PR is too old and should be merged as is.

**Review by houssine78 — APPROVED** [review]

**Comment by houssine78** [comment]
thank you @leemannd do you know who can merge the PR now ?

**Comment by leemannd** [comment]
Hello @houssine78, it is generally done by a maintainter or a PSC member (cf. https://github.com/OCA/timesheet/pull/295#issuecomment-563222138)

**Comment by leemannd** [comment]
Hello @alexey-pelykh can you have a look at it?
Thanks

File: hr_timesheet_activity_begin_end/models/account_analytic_line.py (hunk 5 above)
Comment (alexey-pelykh) [reply hr_timesheet_activity_begin_end/models/account_analytic_line.py]: 💯  "Copyright" instead of "C"

**Review by alexey-pelykh — APPROVED** [review]
Code looks good, but I'm not expert in v9

File: hr_timesheet_activity_begin_end/models/account_analytic_line.py (hunk 5 above)
Comment (leemannd) [reply hr_timesheet_activity_begin_end/models/account_analytic_line.py]: @alexey-pelykh I fully agree witih you. But somehow I find more interesting to have this merged than to block further this old PR.
Thanks for your review.

Comment (alexey-pelykh) [reply hr_timesheet_activity_begin_end/models/account_analytic_line.py]: I say that I'd agree, yet not agree enough to hit merge, and I'd opt in for second PSC member opinion

**Comment by houssine78** [comment]
@alexey-pelykh thanks for your review the PR is from april 2017. Can we just go ahead without asking this minor change if @Martronic-SA don't react ?

**Comment by Martronic-SA** [comment]
@houssine78 We've already waited almost three years so we are not two three days away..
I'm going to chage this now.

**Comment by Martronic-SA** [comment]
Ok I close that PR a make a new one with proper author.

**Pull request closed without merging**